src/connexion/extended/auth_paths_extd.py

```diff
@@ -98,7 +109,89 @@ def add_openapi_yaml(self):
         )
 
 
+class AioHttpApi(AbstractAPI, _AioHttpApi):
+    logger = logging.getLogger('connexion.apis.aiohttp_api')
+
+    def __init__(self, *args, **kwargs):
+        # NOTE we use HTTPPermanentRedirect (308) because
+        # clients sometimes turn POST requests into GET requests
+        # on 301, 302, or 303
+        # see https://tools.ietf.org/html/rfc7538
+        trailing_slash_redirect = normalize_path_middleware(
+            append_slash=True,
+            redirect_class=HTTPPermanentRedirect
+        )
+        self.subapp = web.Application(
+            middlewares=[
+                problems_middleware,
+                trailing_slash_redirect
+            ]
+        )
+        AbstractAPI.__init__(self, *args, **kwargs)
+
+        aiohttp_jinja2.setup(
+            self.subapp,
+            loader=jinja2.FileSystemLoader(
+                str(self.options.openapi_console_ui_from_dir)
+            )
+        )
+        middlewares = self.options.as_dict().get('middlewares', [])
+        self.subapp.middlewares.extend(middlewares)
```

Comment:
Rewriting the entire initialization method to override the `AbstractAPI` initialization inside.